app/models/job.js

```diff
@@ -200,6 +200,24 @@ export default Model.extend(DurationCalculations, {
     }
   }),
 
+  isTrustySudoFalse: Ember.computed('queue', function () {
```

Comment:
This type of thing is so common as a computed property that it exists as [`Ember.computed.equal`](http://emberjs.com/api/classes/Ember.computed.html#method_qqual). You could rewrite it like this to be a bit more concise:

```
isTrustySudoFalse: Ember.computed.equal('queue', 'builds.ec2'),
```